MM-32230: Clarify mmctl vs cli usage in HA (#4326)
mattermost/docs: source/administration/command-line-tools.rst

From the directory where the Mattermost server is installed, a ``mattermost`` command is available for configuring the system. For an overview of the Mattermost command line interface (CLI), `read this article <https://medium.com/@santosjs/plugging-in-to-the-mattermost-cli-8cdcef2bd1f6>`__ from Santos.

.. note::

>   When running `CLI commands <https://docs.mattermost.com/administration/command-line-tools.html>`__ in a `High Availability environment <https://docs.mattermost.com/deployment/cluster.html>`__, performing tasks such as creating and deleting users or changing configuration settings will require a server restart. This is because the CLI is run in a single node which bypasses the mechanisms that a High Availability environment uses to perform actions across all nodes in the cluster. 

Reviewer: ```suggestion
  The CLI is run in a single node which bypasses the mechanisms that a `High Availability environment <https://docs.mattermost.com/deployment/cluster.html>`__ uses to perform actions across all nodes in the cluster. As a result, when running `CLI commands <https://docs.mattermost.com/administration/command-line-tools.html>`__ in a High Availability environment, tasks such as creating and deleting users or changing configuration settings require a server restart.
```
